src/symptom_burst.py

```diff
@@ -0,0 +1,21 @@
+""" Module for defining a series of back to back symptom occurences """
+
+class SymptomBurst:
+    """ Object for defining the collection of symtpoms that occured
+        in succession """
+
+    def __init__(self, symptoms):
+        self.symptoms = symptoms
+        self.burst_start_timestamp = symptoms[0].start_timestamp
+        self.action_msg = symptoms[0].action_msg
+        self.tag = symptoms[0].tag
+        self.symptom_count = len(symptoms)
+
+        duration = None
```

Comment:
Why can't this be initialized as a zero duration, instead of None? Seems like it would simplify the code a little.

Reply:
+1

Reply:
(Realized I never hit send on this comment)

Reply:
It's okay if you want to deliberately distinguish None from a zero duration. But if that's not a meaningful distinction then it can often be better to just always keep the non-None value, so that every single consumer of the data doesn't have to special-case a None check.

Reply:
Good point, I'll make that change.